### DISCUSSION TITLE
Help debugging connection

### DISCUSSION THREAD

**Discussion by kkempter-getswift** [yuezk/GlobalProtect-openconnect#391] [General]
All;

I have spent some time pulling the info from the windows logs for a VPN connection (GlobalProtect) that I want to replicate on my Linux laptop.

I finally am able to connect via the mods I made to the hip report, and I can ping the target jump box behind the vpn. However if I try to ssh to the jump box then ssh just hangs.

Thoughts?

Thanks in advance

**Comment by yuezk** [comment]
Hi @kkempter-getswift can you ssh to the jump box on Windows with the official client?
